kolibri/plugins/facility/assets/src/views/ClassEnrollForm.vue

```diff
@@ -1,6 +1,6 @@
-<template>
+<template xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
```

Comment:
Could you undo this change?

```suggestion
<template>
```